.github/workflows/artifacts.yml

```
@@ -0,0 +1,60 @@
+name: Build artifacts
+on: [ push ]
```

can you check if this should it not be

```suggestion
on: [ push, pull_request ]
```

I have a feeling that otherwise, it will not work on PR's from forks

---

It should and will be, but I have to test that. So I change it to push when I want it to trigger on my fork, then I revert (and squish so the commit history is clear).

---

TBH I was planning to leave it at 
```
on:
  push:
    branches:
      - main
```

So the artifacts are built only on the main branch builds (I don't see any reason for it to be done on PR).

---

I at least two reasons to do it for other branches and PRs as well.

1. We are sure that it will work after merging
2. We can use the artifacts from PR for testing on different environments